operator/config/crd/bases/platform.stackrox.io_securedclusters.yaml

```diff
@@ -398,6 +398,7 @@ spec:
                         - EBPF
                         - CORE_BPF
                         - NoCollection
+                        - KernelModule
```

Comment:
This should have been added to `operator/bundle/manifests/platform.stackrox.io_securedclusters.yaml` too.

That's why the `Style / check-generated-files (pull_request)` check is failing.

Reply:
I think you're not supposed to edit this file manually, but run `make -C operator manifests generate bundle`